### DISCUSSION TITLE
Logarithmic Sliders

### DISCUSSION THREAD

**Discussion by Niroi** [isl-org/Open3D#6313] [Ideas]
Hi,
similar to the ipywidgets slider i would like to have the ability to use a slider with a logarithmic value scale. I need a lot of range of values with the ability to set really small values which is inhibited by the step size of the slider. (Or more just the size of the slider itself) I think a logarithmic slider would be really useful.

Kind regards,
Niori

**Comment by benjaminum** [comment]
Would it be possible to add the `2^x` or `e^x` to the slider value as a workaround?

**Reply by Niroi** [reply]
That would be possible yes, but the slider would always snap to the value of the equation then after letting go. the problem is when i want really low values between 0 and 1, the slider will basically always snap to the left and stay there. 

I think one solution would to just use the slider as a function input and display the result of an equation in a different widget.
Slider with range [0,1000] ----> f(x) = 1/1000 -> display value separate widget. I'm probably sticking to this idea, as it still gives me reasonable control with the slider.

**Reply by benjaminum** [reply]
I think we mean the same thing. You can use a mapping f(x) = 2^x where x is the linear slider value and should you give better control.

**Reply by Niroi** [reply]
Yes correct. I use the slider as input for a function which is displayed in a number edit. I just think it would be nice to display the y from f(x)=y in the slider itself without the slider snapping to that position.

But my current solutions works well enough, so i guess this is really a niche problem that can be adressed as such. :+1:
